### ISSUE TITLE
Loading hdf5 file instead of hub

### ISSUE THREAD

**Issue by Rusiecki** [strongio/keras-elmo#14]
Hey
is there any equivalent to load a pretrained ELMo with ( hdf5 and options.json ) instead of loading tensorflow hub?

what i want to replace is 

`def build(self, input_shape):
        self.elmo = hub.Module('https://tfhub.dev/google/elmo/2', trainable=self.trainable, name="{}_module".format(self.name))
self.trainable_weights += K.tf.trainable_variables(scope="^{}_module/.*".format(self.name))
        super(ElmoEmbeddingLayer, self).build(input_shape)`

with for example something like this

`def build (options ,hdf5):
elmo = allennlp.modules.elmo.Elmo(ELMo_options, ELMo_hdf5File, 2, dropout=0)
return elmo
`

**Comment by jacobzweig** [comment]
Hey @Rusiecki, apologies but you'd need to substantially restructure the existing codebase to do so.
